citus_dev/citus_dev

```diff
@@ -65,9 +64,6 @@ def createNodeCommands(clustername, role, index=None, usessl=False, mx=False):
         run(f"openssl req -new -x509 -days 365 -nodes -text -out {clustername}/{nodename}/server.crt -keyout {clustername}/{nodename}/server.key -subj '/CN={nodename}'")
         run(f"chmod 0600 {clustername}/{nodename}/server.key")
 
-    if mx:
-        run(f"echo \"citus.replication_model = 'streaming'\" >> {clustername}/{nodename}/postgresql.conf")
-
```

Comment:
Is this not a requirement anymore? Or has this setting be made the default now?

Reply:
Yeah, that is the default for a while now